templates/HOWTO.postgresql.conf

```diff
@@ -19,4 +21,6 @@ and save it under the 'templates' role directory
 
 5) If there are new options or some of them removed, update the 'default/main.yml' file and add a "(>= 9.X)" or "(<= 9.X)" comment to them.
 
-6) Update the '.travis.yml' file to test its new version.
+7) For yum based installation add version and minor version of postgresql in 'default/main.yml' under '# YUM settings' at end of file
```

Comment:
You skipped the point 6) (5 to 7)

Reply:
Hi Sébastien,
all ansible tests now pass including a new one for ansible 2.4.2.0/posgresql10.
Docker tests currently commented out - do not know docker ;-).

Reply:
I think that @sebalix was saying that in the file, there is now no step 6

1)
2)
3)
4)
5)
5)  (SHOULD THIS BE STEP 6?)
7)
8)